src/components/viewers/ParquetView.tsx

```diff
@@ -4,6 +4,8 @@ import React, { useCallback, useEffect, useState } from 'react'
 import { RoutesConfig, appendSearchParams } from '../../lib/routes.js'
 import { FileSource } from '../../lib/sources/types.js'
 import { parquetDataFrame } from '../../lib/tableProvider.js'
+import { stringify } from '../../lib/utils.js'
+import classes from '../../styles/ParquetView.module.css'
```

Comment:
nit: would prefer `styles` since classes could also refer to object oriented classes